loader/checkpoint.go

```diff
@@ -205,7 +210,9 @@ func (cp *RemoteCheckPoint) Load(tctx *tcontext.Context) error {
 
 // GetRestoringFileInfo implements CheckPoint.GetRestoringFileInfo
 func (cp *RemoteCheckPoint) GetRestoringFileInfo(db, table string) map[string][]int64 {
-	if tables, ok := cp.restoringFiles[db]; ok {
+	cp.restoringFiles.RLock()
+	defer cp.restoringFiles.RUnlock()
+	if tables, ok := cp.restoringFiles.pos[db]; ok {
```

Comment:
Do we need to return a deep copy (and for `GetAllRestoringFileInfo`)?

Reply:
https://github.com/pingcap/dm/pull/1145/commits/2be83bf8e04ac8a52d0a8b74ee8a9e29b1a57eb7